modules/end-user-guide/partials/proc_creating-a-workspace-from-template-devfile.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+
+[id="creating-a-workspace-from-template-devfile_{context}"]
+= Creating a workspace from template devfile
+
+You can create a new workspace by using one of the devfile templates included in {prod-short}.
+
+.Prerequisites
+
+* A running instance of {prod-short}
```

Comment:
.

Reply:
Period omitted deliberately. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."